### PR TITLE
Update UI configuration

### PR DESCRIPTION
**Pull request by nikcio**
## Proposed changes
This updates the packages and configuration used for the Backoffice UI.

Generally, it wasn't set up in a working way (at least for prettier) and was using outdated versions, which have now been updated.

This updates the eslint setup to use the recommended setup from here: https://typescript-eslint.io/getting-started/legacy-eslint-setup

I've also replaced the a11y eslint rules as it wasn't registered and targeted JSX which isn't in the UI project. Instead, I've added a Lit-specific a11y rule set.

## Notes
- I could see that the existing prettier config was set to tab space `2` and I've also added it to the vs-code settings.json but I generally see `4` as the tab spacing in other projects so it should maybe be set to `4` 🤷 ?
- If this is merged a `npm run lint --fix` and `npm run format` should be run on the project as a whole to align the formatting and fixing the easy eslint rule violations.

***
@Infocaster/developers

### CODE REVIEW

**Comment by D-Inventor** [comment]
Hi @nikcio !! I love this contribution. I'm very happy to see this.

As I'm more backend focussed, this is going over my head a little bit. I will share this with a coworker to get a second opinion.

I see you updated the uui library and angularjs types. I know that for both of those, we rely on the version that Umbraco supplies. I need to confirm that Umbraco's supplied versions match with your update before I merge this. (The uui version was based on what U10 supplied, but I never checked for U13)

Thank you for this, I will get back to you, hopefully today.

File: src/UrlTracker.Backoffice.UI/package.json
Comment (D-Inventor) [inline src/UrlTracker.Backoffice.UI/package.json]: So I checked out the Umbraco source code for Umbraco 13 and found the package.json that Umbraco uses in U13:
https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Web.UI.Client/package.json

It looks like they use uui version 1.7.1 and angular version 1.8.3.
So they should be updated, but I think we should match the versions that Umbraco uses to prevent issues due to version mismatches.

**Review by D-Inventor — CHANGES_REQUESTED** [review]
Love it, just need to match the package versions for angular and umbraco uui to what Umbraco uses internally. That is:
- angular 1.8.3
- umbraco-uui 1.7.1

UPDATE: My coworker has also checked and confirmed that this is all cool, so if above is resolved, then it can be merged

**Comment by nikcio** [comment]
@D-Inventor The PR has been updated 😄

**Review by D-Inventor — APPROVED** [review]

**Pull request merged**
